scipy/stats/_qmc.py

```diff
@@ -1035,6 +1035,23 @@ class LatinHypercube(QMCEngine):
         Dimension of the parameter space.
     centered : bool, optional
         Center the point within the multi-dimensional grid. Default is False.
+
+        .. deprecated:: 1.10.0
+            `centered` is deprecated as of SciPy 1.10.0 and will be removed in
+            1.11.0. Use `scramble` instead. ``centered=True`` corresponds to
+            ``scramble=False``.
+
+    scramble : bool, optional
+        When False, center the point within the multi-dimensional grid.
+
+        .. note::
+            Not scrambling will not make the sample constant. For that,
```

Comment:
"constant" is somewhat ambiguous. I would specify "Setting `scramble=False` does not ensure deterministic output. For that, fix the `seed` parameter."

Maybe specify what `scramble=True` does.

Reply:
I rephrased, LMK